webcompat/static/css/src/issue-wizard.css

```diff
@@ -654,6 +655,10 @@
   width: 28px;
 }
 
+.issue-form .is-validated .text-field {
+  border: 1px solid var(--issue-selected-outline);
+}
```

Comment:
I think here we can only change border-color (since the border is already 1px solid )